### PR TITLE
Update gluster setup readme.

### PR DESCRIPTION
**Pull request by humblec**
Referring gluster setup guide to official glusterfs documentation. 

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/36715)

### CODE REVIEW

**Comment by dims** [comment]
@k8s-bot ok to test

File: examples/volumes/glusterfs/README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ The example assumes that you have already set up a Glusterfs server cluster and
 
 ### Prerequisites
 
-Set up Glusterfs server cluster; install Glusterfs client package on the Kubernetes nodes. ([Guide](https://www.howtoforge.com/high-availability-storage-with-glusterfs-3.2.x-on-debian-wheezy-automatic-file-replication-mirror-across-two-storage-servers))
+Set up Glusterfs server cluster; install Glusterfs client package on the Kubernetes nodes. ([Guide](http://gluster.readthedocs.io/en/latest/Administrator%20Guide/))
```
Comment (humblec) [inline examples/volumes/glusterfs/README.md]: @rootfs The gluster documentation is in gluster.readthedocs.io and its getting rendered from https://github.com/gluster/glusterdocs . The gluster.org points to gluster.readthedocs.io.

**Comment by rootfs** [comment]
@dims can you remove do-not-merge?

**Comment by dims** [comment]
@rootfs done

**Comment by rootfs** [comment]
@dims thanks!

**Pull request merged**